zellij-utils/src/kdl/kdl_layout_parser.rs

```diff
@@ -334,6 +334,13 @@ impl<'a> KdlLayoutParser<'a> {
     fn parse_cwd(&self, kdl_node: &KdlNode) -> Result<Option<PathBuf>, ConfigError> {
         Ok(
             kdl_get_string_property_or_child_value_with_error!(kdl_node, "cwd")
+                .and_then(|cwd| match shellexpand::env(cwd) {
+                    Ok(cwd) => Some(cwd.to_string()),
+                    Err(e) => {
+                        log::error!("when parsing `cwd`: {}", e);
```

Comment:
Let's return a proper error here so that it can be displayed nicely to the user. You can look at the various places we return a `ConfigError` in this file for examples.

Reply:
Will do. I did it this way because of how it behaved before the change, but an error is indeed more reasonable.